src/utils.js

```diff
@@ -846,7 +846,21 @@ export function getBidderRequest(bidRequests, bidder, adUnitCode) {
       .filter(bid => bid.bidder === bidder && bid.adUnitCode === adUnitCode).length > 0;
   }) || { start: null, auctionId: null };
 }
-
+/**
+ * Returns user configured bidder params from adunit
+ * @param {object} adunits
+ * @param {string} adunit code
+ * @param {string} bidder code
+ * @return {Array} user configured param for the given bidder adunit configuration
```

Comment:
it's really hard to visulize what this object looks like. Can you add it to the jsdocs typedef here?

Reply:
return type is Array of Objects, 
```javascript
const adUnits = [{
      code: 'adUnit1',
      bids: [{
        bidder: 'bidder1',
        params: {
          key1: 'value1'
        }
      }]
    }];
```
In this case object is adUnits[0].params, which is defined by user